frontend/src/views/Blacksmith.vue

```diff
@@ -1680,10 +1681,10 @@ export default Vue.extend({
 
 .blacksmith-page{
   background-image: url('../assets/blacksmith/cb-reforge-bg.jpg');
-  background-size: cover;
+  background-size: clamp(100%, 100%, 100%) auto;
   background-repeat: no-repeat;
```

Comment:
Maybe cover is better here?
![image](https://user-images.githubusercontent.com/5601589/189245990-450709ad-e431-4efa-acbc-e2982e32e396.png)